Sdext.tex

```diff
@@ -149,6 +149,21 @@ \section{Reset}
 initialization that would usually happen before the first instruction is
 executed.
 
+\section{Halt}
+
+\begin{steps}{When a hart halts:}
+    \item \FcsrDcsrCause is updated.
+    \item \FcsrDcsrPrv and \FcsrDcsrV are set to reflect current privilege mode.
+    \item \RcsrDpc is set to the next instruction that should be executed.
+    \item If the current instruction can be partially executed and should be
+        restarted to complete, then the relevant state for that is updated. E.g.
+        if a halt occurs during a partially executed vector instruction, then
+        {\tt vstart} is updated, and \RcsrDpc is updated to the address of the
+        partially executed instruction. This is analogous to how vector
+        instruction behave for other exceptions.
```

Comment:
While I think of halting as an exception, it's technically something different.  So removing the word "other" would keep the spirit of what you're saying and prevent an email question later.